setup.py

```diff
@@ -1,12 +1,12 @@
 from setuptools import setup
 
 setup(name='cartopy_offlinedata',
-      version='0.2.2',
+      version='0.2.3',
```

Comment:
I bumped the version number, assuming we can do a release if you're good with this change.